main.go

```diff
@@ -41,6 +43,7 @@ func init() {
 	rootCmd.PersistentFlags().BoolVarP(&dryrun, "dryrun", "n", false, "Do nothing destructive, only print")
 	rootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "Be more verbose")
 	rootCmd.PersistentFlags().BoolVarP(&concurrent, "concurrent", "c", false, "Allow more than one zfs-cleaner to operate on the same configuration file simultaneously")
+	rootCmd.PersistentFlags().BoolVarP(&plancheck, "plancheck", "p", false, "Print mounts that have no plan and exit")
```

Comment:
Is this really a flag? Isn't it more like a subcommand? You can't combine this flag with anything else.